homeassistant/components/unifiprotect/sensor.py

```diff
@@ -0,0 +1,408 @@
+        key="storage_used",
+        name="Storage Used",
+        native_unit_of_measurement=DATA_BYTES,
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        ufp_value="stats.storage.used",
+    ),
+    ProtectSensorEntityDescription(
+        key="write_rate",
+        name="Disk Write Rate",
+        native_unit_of_measurement=DATA_RATE_BYTES_PER_SECOND,
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        ufp_value="stats.storage.rate",
+        precision=2,
+    ),
+    ProtectSensorEntityDescription(
+        key="voltage",
+        name="Voltage",
+        device_class=DEVICE_CLASS_VOLTAGE,
+        native_unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        ufp_value="voltage",
+        # no feature flag, but voltage will be null if device does not have voltage sensor
+        # (i.e. is not G4 Doorbell or not on 1.20.1+)
+        ufp_required_field="voltage",
+        precision=2,
+    ),
+)
+
+SENSE_SENSORS: tuple[ProtectSensorEntityDescription, ...] = (
+    ProtectSensorEntityDescription(
+        key="battery_level",
+        name="Battery Level",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=DEVICE_CLASS_BATTERY,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        ufp_value="battery_status.percentage",
+    ),
+    ProtectSensorEntityDescription(
+        key="light_level",
+        name="Light Level",
+        native_unit_of_measurement="lx",
+        device_class=DEVICE_CLASS_ILLUMINANCE,
+        ufp_value="stats.light.value",
+    ),
+    ProtectSensorEntityDescription(
+        key="humidity_level",
+        name="Humidity Level",
+        native_unit_of_measurement="%",
+        device_class=DEVICE_CLASS_HUMIDITY,
+        ufp_value="stats.humidity.value",
+    ),
+    ProtectSensorEntityDescription(
+        key="temperature_level",
+        name="Temperature",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        ufp_value="stats.temperature.value",
+    ),
+)
+
+NVR_SENSORS: tuple[ProtectSensorEntityDescription, ...] = (
+    ProtectSensorEntityDescription(
+        key="uptime",
+        name="Uptime",
+        icon="mdi:clock",
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+        ufp_value="up_since",
+    ),
+    ProtectSensorEntityDescription(
+        key="cpu_utilization",
+        name="CPU Utilization",
+        native_unit_of_measurement=PERCENTAGE,
+        icon="mdi:speedometer",
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        ufp_value="system_info.cpu.average_load",
+    ),
+    ProtectSensorEntityDescription(
+        key="cpu_temperature",
```

Comment:
Either all of the keys should be constants or none of them.  I don't think there is any good reason that they need to be constants since they are only used once